### PR TITLE
Fritz deprecate services for button entities

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Inform users that services will be soon deprecated in favor of new button entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/61483
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fritz.markdown [hunk 1]
```diff
@@ -76,6 +76,8 @@ A device is identified as stale when it's still present on Home Assistant but no
 | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
 | `entity_id`            | no       | Only act on a specific  router                                                                                 |
 
+**Note**: Services will be soon deprecated in favor of button entities, that have the same functionality.
```
Comment (frenck) [inline source/_integrations/fritz.markdown]: Remove the services from the documentation instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fritz.markdown (hunk 1 above)
Comment (chemelli74) [reply source/_integrations/fritz.markdown]: Done, sorry for the late reply.

Comment (frenck) [reply source/_integrations/fritz.markdown]: Don't worry, there is always time 😄

**Review by frenck — APPROVED** [review]
Thanks, @chemelli74 👍

**Pull request merged**